### PR TITLE
Update packaging script to match electron-packager breaking changes

### PR DESCRIPTION
**Pull request by jbleuzen**
With version 8.0.0 of electron-packager asar options changed and you don't need --asar when you are using --asar.unpack.

This fix the behaviour of the "View demo" button on Protocol Handler demo.

### CODE REVIEW

**Comment by zeke** [comment]
Thanks, @jbleuzen. This looks strange though:

```
--asar.unpack
```

Sure you got that right?

**Comment by zeke** [comment]
cc @malept 👀

**Comment by jbleuzen** [comment]
See the remove section :
https://github.com/electron-userland/electron-packager/compare/v7.7.0...v8.0.0

**Comment by zeke** [comment]
Ah I see it here: https://github.com/electron-userland/electron-packager/blob/5bbbf03b5887bc25939b50858122d0df30d868c0/NEWS.md#removed

It looks like `news.md` and `docs/api.md` have not yet been updated to reflect what's in version 8

**Comment by jbleuzen** [comment]
There is also this line explaining that you must not combine both `--asar` and `--asar.unpack`

https://github.com/electron-userland/electron-packager/blob/5bbbf03b5887bc25939b50858122d0df30d868c0/usage.txt#L30

File: package.json [hunk 1]
```diff
@@ -12,9 +12,9 @@
...
-    "pack-linux": "electron-packager . --asar --asar-unpack=protocol-link.html --overwrite --platform=linux --arch=x64 --icon=assets/app-icon/png/64.png --prune=true --out=out",
+    "pack-mac": "electron-packager . --asar.unpack=protocol-link.html --overwrite --platform=darwin --arch=x64 --icon=assets/app-icon/mac/app.icns --prune=true --out=out --osx-sign.identity='Developer ID Application: GitHub' --extend-info=assets/mac/info.plist",
+    "pack-win": "electron-packager . ElectronAPIDemos --asar.unpack=protocol-link.html --overwrite --platform=win32 --arch=ia32 --icon=assets/app-icon/win/app.ico --prune=true --out=out --version-string.CompanyName='GitHub, Inc.' --version-string.FileDescription='Electron API Demos' --version-string.ProductName='Electron API Demos'",
+    "pack-linux": "electron-packager . --asar.unpack=protocol-link.html --overwrite --platform=linux --arch=x64 --icon=assets/app-icon/png/64.png --prune=true --out=out",
```
Comment (malept) [inline package.json]: --icon is not necessary for Linux, see API docs for icon

**Comment by malept** [comment]
> It looks like news.md and docs/api.md have not yet been updated to reflect what's in version 8

Huh?

**Comment by zeke** [comment]
Oops I wasn't looking at latest master. Sorry for the confusion and thanks for the fix, @jbleuzen

**Pull request merged**